cms/static/js/views/previous_video_upload.js

```diff
@@ -29,11 +30,21 @@ define(
                         videoImageSettings: options.videoImageSettings
                     });
                 }
+                if (this.isVideoTranscriptEnabled) {
```

Comment:
Why we need this check?

Reply:
transcripts column would only be shown when transcripts feature is enabled otherwise not.